buildpipeline/Core-Setup-Publish.json

```diff
@@ -383,7 +383,7 @@
       "allowOverride": true
     },
     "NUGET_FEED_URL": {
-      "value": "https:%2F%2Fdotnet.myget.org/F/dotnet-core/api/v2/package"
+      "value": "https:%2F%2Fdotnetmygetlegacy.blob.core.windows.net/dotnet-core/index.json"
```

Comment:
I think this should be: https://pkgs.dev.azure.com/dnceng/public/_packaging/myget-legacy/nuget/v3/index.json

Reply:
The dotnet-core feed from MyGet was moved to that sleet feed. It was the only MyGet feed not moved to Azure Artifacts.